docs/features/techdocs/README.md

```diff
@@ -66,6 +66,7 @@ See [TechDocs Architecture](architecture.md) to get an overview of where the bel
 | GitHub Enterprise            | Yes ✅         |
 | Bitbucket                    | Yes ✅         |
 | Azure DevOps                 | Yes ✅         |
+| Gerrit                       | Yes ✅         |
```

Comment:
Hi @anicke, thank you for being part of our team of contributors 🙌🏻 !
May I ask you to also add an example of how to use Gerrit with TechDocs here?

Reply:
Hi, I don't know what I could add really. From a backstage perspective it works just like the other integrations (github etc).

Reply:
Looks like there's plenty gerrit-related config examples elsewhere in docs, so I'm good with this as is.  Thanks @anicke!